hub/core/dataset.py

```diff
@@ -214,8 +219,45 @@ def create_tensor(
 
         tensor.info.update(info_kwargs)
 
+        if hash_samples:
+            hashes_tensor = self.create_tensor("hashes", htype = htype, sample_compression=sample_compression)
```

Comment:
don't copy the htype and sample_compression. the hashes tensor should have this predefined